### ISSUE TITLE
quicklisp-extra-libs cloned into wrong location

### ISSUE THREAD

**Issue by tviti** [atlas-engineer/nyxt#815]
Follow up from https://github.com/atlas-engineer/nyxt/pull/807 (I'm reluctant to re-open the PR, since I just realized I submitted w/o out branching).

I no longer have to manually re-clone quicklisp-client, but it looks like quicklisp-extra-libs still isn't getting cloned into the correct location (or depending on how you look at it, the symlinks aren't pointing to the correct location). Here's what I get trying to build from a fresh clone:

```
[nix-shell:~/Source/nyxt]$ PAGER=cat git log -n 1
commit e6ab4858d7851dc344766a214bf4ca5c96418bfa (HEAD -> master, origin/master, origin/HEAD)
Author: John Mercouris <john@mercouris.email>
Date:   Thu Jun 25 18:16:27 2020 +0200

    bookmark.lisp: fix insert-suggestion-or-tag

[nix-shell:~/Source/nyxt]$ make all
make application
make[1]: Entering directory '/mnt/data/tviti/Source/nyxt'
true && make build-deps || true
make[2]: Entering directory '/mnt/data/tviti/Source/nyxt'
true && git submodule update --init --remote || true
Submodule 'quicklisp-client' (https://github.com/quicklisp/quicklisp-client) registered for path 'quicklisp-client'
Submodule 'quicklisp/local-projects/cl-webkit' (https://github.com/joachifm/cl-webkit) registered for path 'quicklisp-libraries/cl-webkit'
Submodule 'quicklisp/local-projects/cluffer' (git@github.com:robert-strandh/Cluffer.git) registered for path 'quicklisp-libraries/cluffer'
Cloning into '/mnt/data/tviti/Source/nyxt/quicklisp-client'...
Cloning into '/mnt/data/tviti/Source/nyxt/quicklisp-libraries/cl-webkit'...
Cloning into '/mnt/data/tviti/Source/nyxt/quicklisp-libraries/cluffer'...
Submodule path 'quicklisp-client': checked out 'b525ae5a4db984bf96a33b9c7379b3b13a4edd98'
Submodule path 'quicklisp-libraries/cl-webkit': checked out 'c023dad817e8ff031073bf2c76119707e73992a4'
Submodule path 'quicklisp-libraries/cluffer': checked out '4aad29c276a58a593064e79972ee4d77cae0af4a'
for i in quicklisp-libraries/local-projects/*; do ln -sf "$i" "quicklisp-client/local-projects/$(basename "$i")"; done
ln: failed to create symbolic link 'quicklisp-client/local-projects/*': No such file or directory
make[2]: *** [Makefile:109: quicklisp-extra-libs] Error 1
make[2]: Leaving directory '/mnt/data/tviti/Source/nyxt'
env NYXT_INTERNAL_QUICKLISP=true sbcl --no-userinit --non-interactive \
	--eval '(require "asdf")' \
	--eval '(when (string= (uiop:getenv "NYXT_INTERNAL_QUICKLISP") "true") (load "quicklisp-client/setup.lisp"))' \
	--load nyxt.asd \
	--eval '(asdf:make :nyxt/gtk-application)' \
	--eval '(uiop:quit)' || (printf "\n%s\n%s\n" "Compilation failed, see the above stacktrace." && exit 1)
This is SBCL 2.0.0.nixos, an implementation of ANSI Common Lisp.
More information about SBCL is available at <http://www.sbcl.org/>.

SBCL is free software, provided as is, with absolutely no warranty.
It is mostly in the public domain; some portions are provided under
BSD-style licenses.  See the CREDITS and COPYING files in the
distribution for more information.
; Fetching #<URL "http://beta.quicklisp.org/dist/quicklisp.txt">
; 0.40KB
==================================================
408 bytes in 0.00 seconds (398.44KB/sec)
Installing dist "quicklisp" version "2020-06-10".
; Fetching #<URL "http://beta.quicklisp.org/dist/quicklisp/2020-06-10/releases.txt">
; 456.86KB
==================================================
467,822 bytes in 0.31 seconds (1493.00KB/sec)
; Fetching #<URL "http://beta.quicklisp.org/dist/quicklisp/2020-06-10/systems.txt">
; 334.64KB
==================================================
342,672 bytes in 0.27 seconds (1221.32KB/sec)
Unhandled ASDF/FIND-COMPONENT:MISSING-DEPENDENCY in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                                              {10005184C3}>:
  Component :ALEXANDRIA not found, required by #<SYSTEM "nyxt">

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {10005184C3}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK Component :ALEXANDRIA not found, required by #<SYSTEM "nyxt"> #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK *INVOKE-DEBUGGER-HOOK* Component :ALEXANDRIA not found, required by #<SYSTEM "nyxt">)
2: (INVOKE-DEBUGGER Component :ALEXANDRIA not found, required by #<SYSTEM "nyxt">)
3: (ERROR ASDF/FIND-COMPONENT:MISSING-DEPENDENCY :REQUIRED-BY #<ASDF/SYSTEM:SYSTEM "nyxt"> :REQUIRES :ALEXANDRIA)

<further output snipped>
```

Not sure but I think the last error about not finding alexandria is unrelated. I can get through it if I build a second time, but then I run into the same error I saw previously about the local libs not being found. Here's the log from trying to build a third time on the fresh clone (since I figure you probably don't want to see the package fetch/build messages):

```
[nix-shell:~/Source/nyxt]$ make all
make application
make[1]: Entering directory '/mnt/data/tviti/Source/nyxt'
true && make build-deps || true
make[2]: Entering directory '/mnt/data/tviti/Source/nyxt'
true && git submodule update --init --remote || true
for i in quicklisp-libraries/local-projects/*; do ln -sf "$i" "quicklisp-client/local-projects/$(basename "$i")"; done
sbcl --no-userinit --non-interactive \
	--eval '(require "asdf")' \
	--load quicklisp-client/setup.lisp \
	--load nyxt.asd \
	--eval '(ql:quickload :nyxt/gtk-application)' \
	--eval '(uiop:quit)' || true
This is SBCL 2.0.0.nixos, an implementation of ANSI Common Lisp.
More information about SBCL is available at <http://www.sbcl.org/>.

SBCL is free software, provided as is, with absolutely no warranty.
It is mostly in the public domain; some portions are provided under
BSD-style licenses.  See the CREDITS and COPYING files in the
distribution for more information.
To load "nyxt/gtk-application":
  Load 1 ASDF system:
    nyxt/gtk-application
; Loading "nyxt/gtk-application"
.....Unhandled QUICKLISP-CLIENT:SYSTEM-NOT-FOUND in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                                         {10005184C3}>:
  System "cluffer" not found

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {10005184C3}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK #<QUICKLISP-CLIENT:SYSTEM-NOT-FOUND {100159A683}> #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK *INVOKE-DEBUGGER-HOOK* #<QUICKLISP-CLIENT:SYSTEM-NOT-FOUND {100159A683}>)
2: (INVOKE-DEBUGGER #<QUICKLISP-CLIENT:SYSTEM-NOT-FOUND {100159A683}>)
3: (CERROR "Try again" QUICKLISP-CLIENT:SYSTEM-NOT-FOUND :NAME "cluffer")
4: ((LABELS QUICKLISP-CLIENT::RECURSE :IN QUICKLISP-CLIENT::COMPUTE-LOAD-STRATEGY) "cluffer")
5: (QL-DIST::CALL-WITH-CONSISTENT-DISTS #<CLOSURE (LAMBDA NIL :IN QUICKLISP-CLIENT::COMPUTE-LOAD-STRATEGY) {100158FCAB}>)
6: (QUICKLISP-CLIENT::COMPUTE-LOAD-STRATEGY "cluffer")
7: (QUICKLISP-CLIENT::AUTOLOAD-SYSTEM-AND-DEPENDENCIES "cluffer" :PROMPT NIL)
8: (QUICKLISP-CLIENT::AUTOLOAD-SYSTEM-AND-DEPENDENCIES "nyxt/gtk-application" :PROMPT NIL)
9: ((:METHOD QL-IMPL-UTIL::%CALL-WITH-QUIET-COMPILATION (T T)) #<unused argument> #<CLOSURE (FLET QUICKLISP-CLIENT::QL :IN QUICKLISP-CLIENT:QUICKLOAD) {100382059B}>) [fast-method]
10: ((:METHOD QL-IMPL-UTIL::%CALL-WITH-QUIET-COMPILATION :AROUND (QL-IMPL:SBCL T)) #<QL-IMPL:SBCL {1002EAC7D3}> #<CLOSURE (FLET QUICKLISP-CLIENT::QL :IN QUICKLISP-CLIENT:QUICKLOAD) {100382059B}>) [fast-method]
11: ((:METHOD QUICKLISP-CLIENT:QUICKLOAD (T)) :NYXT/GTK-APPLICATION :PROMPT NIL :SILENT NIL :VERBOSE NIL) [fast-method]
12: (QL-DIST::CALL-WITH-CONSISTENT-DISTS #<CLOSURE (LAMBDA NIL :IN QUICKLISP-CLIENT:QUICKLOAD) {100380445B}>)
13: (SB-INT:SIMPLE-EVAL-IN-LEXENV (QUICKLISP-CLIENT:QUICKLOAD :NYXT/GTK-APPLICATION) #<NULL-LEXENV>)
14: (EVAL (QUICKLISP-CLIENT:QUICKLOAD :NYXT/GTK-APPLICATION))
15: (SB-IMPL::PROCESS-EVAL/LOAD-OPTIONS ((:EVAL . "(require \"asdf\")") (:LOAD . "quicklisp-client/setup.lisp") (:LOAD . "nyxt.asd") (:EVAL . "(ql:quickload :nyxt/gtk-application)") (:EVAL . "(uiop:quit)") (:QUIT)))
16: (SB-IMPL::TOPLEVEL-INIT)
17: ((FLET SB-UNIX::BODY :IN SAVE-LISP-AND-DIE))
18: ((FLET "WITHOUT-INTERRUPTS-BODY-7" :IN SAVE-LISP-AND-DIE))
19: ((LABELS SB-IMPL::RESTART-LISP :IN SAVE-LISP-AND-DIE))

unhandled condition in --disable-debugger mode, quitting
make quicklisp-update
make[3]: Entering directory '/mnt/data/tviti/Source/nyxt'
true && sbcl --no-userinit --non-interactive \
	--load quicklisp-client/setup.lisp \
	--eval '(require "asdf")' \
	--eval '(ql:update-dist "quicklisp" :prompt nil)' \
	--eval '(uiop:quit)' || true
This is SBCL 2.0.0.nixos, an implementation of ANSI Common Lisp.
More information about SBCL is available at <http://www.sbcl.org/>.

SBCL is free software, provided as is, with absolutely no warranty.
It is mostly in the public domain; some portions are provided under
BSD-style licenses.  See the CREDITS and COPYING files in the
distribution for more information.
You already have the latest version of "quicklisp": 2020-06-10.
make[3]: Leaving directory '/mnt/data/tviti/Source/nyxt'
make[2]: Leaving directory '/mnt/data/tviti/Source/nyxt'
env NYXT_INTERNAL_QUICKLISP=true sbcl --no-userinit --non-interactive \
	--eval '(require "asdf")' \
	--eval '(when (string= (uiop:getenv "NYXT_INTERNAL_QUICKLISP") "true") (load "quicklisp-client/setup.lisp"))' \
	--load nyxt.asd \
	--eval '(asdf:make :nyxt/gtk-application)' \
	--eval '(uiop:quit)' || (printf "\n%s\n%s\n" "Compilation failed, see the above stacktrace." && exit 1)
This is SBCL 2.0.0.nixos, an implementation of ANSI Common Lisp.
More information about SBCL is available at <http://www.sbcl.org/>.

SBCL is free software, provided as is, with absolutely no warranty.
It is mostly in the public domain; some portions are provided under
BSD-style licenses.  See the CREDITS and COPYING files in the
distribution for more information.
WARNING: System definition file #P"/mnt/data/tviti/Source/nyxt/quicklisp-client/dists/quicklisp/software/cl-json-20141217-git/cl-json.asd" contains definition for system "cl-json.test". Please only define "cl-json" and secondary systems with a name starting with "cl-json/" (e.g. "cl-json/test") in that file.
WARNING: System definition file #P"/mnt/data/tviti/Source/nyxt/quicklisp-client/dists/quicklisp/software/cl-ppcre-20190521-git/cl-ppcre.asd" contains definition for system "cl-ppcre-test". Please only define "cl-ppcre" and secondary systems with a name starting with "cl-ppcre/" (e.g. "cl-ppcre/test") in that file.
WARNING: System definition file #P"/mnt/data/tviti/Source/nyxt/quicklisp-client/dists/quicklisp/software/cl-ppcre-20190521-git/cl-ppcre-unicode.asd" contains definition for system "cl-ppcre-unicode-test". Please only define "cl-ppcre-unicode" and secondary systems with a name starting with "cl-ppcre-unicode/" (e.g. "cl-ppcre-unicode/test") in that file.
WARNING: System definition file #P"/mnt/data/tviti/Source/nyxt/quicklisp-client/dists/quicklisp/software/s-xml-20150608-git/s-xml.asd" contains definition for system "s-xml.test". Please only define "s-xml" and secondary systems with a name starting with "s-xml/" (e.g. "s-xml/test") in that file.
WARNING: System definition file #P"/mnt/data/tviti/Source/nyxt/quicklisp-client/dists/quicklisp/software/s-xml-20150608-git/s-xml.asd" contains definition for system "s-xml.examples". Please only define "s-xml" and secondary systems with a name starting with "s-xml/" (e.g. "s-xml/test") in that file.
WARNING: System definition file #P"/mnt/data/tviti/Source/nyxt/quicklisp-client/dists/quicklisp/software/flexi-streams-20200610-git/flexi-streams.asd" contains definition for system "flexi-streams-test". Please only define "flexi-streams" and secondary systems with a name starting with "flexi-streams/" (e.g. "flexi-streams/test") in that file.
WARNING: System definition file #P"/mnt/data/tviti/Source/nyxt/quicklisp-client/dists/quicklisp/software/cl-fad-20200610-git/cl-fad.asd" contains definition for system "cl-fad-test". Please only define "cl-fad" and secondary systems with a name starting with "cl-fad/" (e.g. "cl-fad/test") in that file.
WARNING: System definition file #P"/mnt/data/tviti/Source/nyxt/quicklisp-client/dists/quicklisp/software/cl-base64-20150923-git/cl-base64.asd" contains definition for system "cl-base64-tests". Please only define "cl-base64" and secondary systems with a name starting with "cl-base64/" (e.g. "cl-base64/test") in that file.
Unhandled ASDF/FIND-COMPONENT:MISSING-DEPENDENCY in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                                              {10005184C3}>:
  Component :CLUFFER not found, required by #<SYSTEM "nyxt/text-buffer">

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {10005184C3}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK Component :CLUFFER not found, required by #<SYSTEM "nyxt/text-buffer"> #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK *INVOKE-DEBUGGER-HOOK* Component :CLUFFER not found, required by #<SYSTEM "nyxt/text-buffer">)
2: (INVOKE-DEBUGGER Component :CLUFFER not found, required by #<SYSTEM "nyxt/text-buffer">)
3: (ERROR ASDF/FIND-COMPONENT:MISSING-DEPENDENCY :REQUIRED-BY #<ASDF/SYSTEM:SYSTEM "nyxt/text-buffer"> :REQUIRES :CLUFFER)
4: (ASDF/FIND-COMPONENT:RESOLVE-DEPENDENCY-NAME #<ASDF/SYSTEM:SYSTEM "nyxt/text-buffer"> :CLUFFER NIL)
5: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/text-buffer"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {100185199B}>)
6: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
7: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
8: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/text-buffer"> NIL)
9: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/text-buffer" "package"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {10018515DB}>)
10: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
11: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
12: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/text-buffer" "package"> NIL)
13: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/text-buffer" "package"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {10018511DB}>)
14: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
15: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
16: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/text-buffer" "package"> NIL)
17: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/text-buffer"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1001850D6B}>)
18: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
19: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
20: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/text-buffer"> NIL)
21: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/SYSTEM:SYSTEM "nyxt"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD3F6B}>)
22: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
23: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
24: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/SYSTEM:SYSTEM "nyxt"> NIL)
25: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt" "patches/patch-serialization"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD319B}>)
26: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
27: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
28: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt" "patches/patch-serialization"> NIL)
29: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt" "patches/patch-serialization"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD2D9B}>)
30: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
31: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
32: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt" "patches/patch-serialization"> NIL)
33: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD28EB}>)
34: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
35: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
36: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt"> NIL)
37: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD248B}>)
38: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
39: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
40: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk"> NIL)
41: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/gtk" "renderer-gtk"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD20EB}>)
42: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
43: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
44: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/gtk" "renderer-gtk"> NIL)
45: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/gtk" "renderer-gtk"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD1D0B}>)
46: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
47: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
48: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/LISP-ACTION:CL-SOURCE-FILE "nyxt/gtk" "renderer-gtk"> NIL)
49: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD18BB}>)
50: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
51: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
52: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk"> NIL)
53: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD0D9B}>)
54: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
55: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
56: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:PREPARE-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application"> NIL)
57: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD0A3B}>)
58: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
59: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
60: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/LISP-ACTION:LOAD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application"> NIL)
61: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/BUNDLE:PROGRAM-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CD01DB}>)
62: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
63: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
64: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/BUNDLE:PROGRAM-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application"> T)
65: (ASDF/PLAN:MAP-DIRECT-DEPENDENCIES #<ASDF/OPERATE:BUILD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application"> #<CLOSURE (LAMBDA (ASDF/PLAN::O ASDF/PLAN::C) :IN ASDF/PLAN:TRAVERSE-ACTION) {1003CC819B}>)
66: ((LAMBDA NIL :IN ASDF/PLAN:TRAVERSE-ACTION))
67: ((LAMBDA NIL :IN ASDF/ACTION:CALL-WHILE-VISITING-ACTION))
68: (ASDF/PLAN:TRAVERSE-ACTION #<ASDF/PLAN:SEQUENTIAL-PLAN {1003CC6CE3}> #<ASDF/OPERATE:BUILD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application"> T)
69: ((LAMBDA NIL :IN ASDF/PLAN:MAKE-PLAN))
70: ((:METHOD ASDF/OPERATE:OPERATE (ASDF/OPERATION:OPERATION ASDF/COMPONENT:COMPONENT)) #<ASDF/OPERATE:BUILD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application"> :PLAN-CLASS NIL :PLAN-OPTIONS NIL) [fast-method]
71: ((SB-PCL::EMF ASDF/OPERATE:OPERATE) #<unused argument> #<unused argument> #<ASDF/OPERATE:BUILD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application">)
72: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
73: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) #<ASDF/OPERATE:BUILD-OP > #<ASDF/SYSTEM:SYSTEM "nyxt/gtk-application">) [fast-method]
74: ((SB-PCL::EMF ASDF/OPERATE:OPERATE) #<unused argument> #<unused argument> ASDF/OPERATE:BUILD-OP :NYXT/GTK-APPLICATION)
75: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
76: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) ASDF/OPERATE:BUILD-OP :NYXT/GTK-APPLICATION) [fast-method]
77: (ASDF/SESSION:CALL-WITH-ASDF-SESSION #<CLOSURE (LAMBDA NIL :IN ASDF/OPERATE:OPERATE) {1003A81E8B}> :OVERRIDE T :KEY NIL :OVERRIDE-CACHE T :OVERRIDE-FORCING NIL)
78: ((LAMBDA NIL :IN ASDF/OPERATE:OPERATE))
79: (ASDF/SESSION:CALL-WITH-ASDF-SESSION #<CLOSURE (LAMBDA NIL :IN ASDF/OPERATE:OPERATE) {100380D2DB}> :OVERRIDE NIL :KEY NIL :OVERRIDE-CACHE NIL :OVERRIDE-FORCING NIL)
80: ((:METHOD ASDF/OPERATE:OPERATE :AROUND (T T)) ASDF/OPERATE:BUILD-OP :NYXT/GTK-APPLICATION) [fast-method]
81: (ASDF/OPERATE:MAKE :NYXT/GTK-APPLICATION)
82: (SB-INT:SIMPLE-EVAL-IN-LEXENV (ASDF/OPERATE:MAKE :NYXT/GTK-APPLICATION) #<NULL-LEXENV>)
83: (EVAL (ASDF/OPERATE:MAKE :NYXT/GTK-APPLICATION))
84: (SB-IMPL::PROCESS-EVAL/LOAD-OPTIONS ((:EVAL . "(require \"asdf\")") (:EVAL . "(when (string= (uiop:getenv \"NYXT_INTERNAL_QUICKLISP\") \"true\") (load \"quicklisp-client/setup.lisp\"))") (:LOAD . "nyxt.asd") (:EVAL . "(asdf:make :nyxt/gtk-application)") (:EVAL . "(uiop:quit)") (:QUIT)))
85: (SB-IMPL::TOPLEVEL-INIT)
86: ((FLET SB-UNIX::BODY :IN SAVE-LISP-AND-DIE))
87: ((FLET "WITHOUT-INTERRUPTS-BODY-7" :IN SAVE-LISP-AND-DIE))
88: ((LABELS SB-IMPL::RESTART-LISP :IN SAVE-LISP-AND-DIE))

unhandled condition in --disable-debugger mode, quitting

Compilation failed, see the above stacktrace.

make[1]: *** [Makefile:43: application] Error 1
make[1]: Leaving directory '/mnt/data/tviti/Source/nyxt'
make: *** [Makefile:39: nyxt] Error 2
```

**Comment by Ambrevar** [comment]
Thanks for insisting! :)
Indeed, I had failed to reproduce this because the deps were already in
my ~/common-lisp folder :/

Should be fixed with d8bcfd2b402dabdbe587d67d81df8fc39711928f.
I've opted for `readlink -f` which should be portable enough.

**Comment by tviti** [comment]
Works now! (`all` target builds successfully; was able to open resulting binary)

> Thanks for insisting! :)

Thanks for working on Nyxt!
